### ISSUE TITLE
How to find the company and the manager

### ISSUE THREAD

**Issue by longandfu** [scanny/python-pptx#586]
prs = Presentation()
core_properties = prs.core_properties

I find author and title and so on, Excuse me,How to find the company and the manager。

**Comment by Sustoja** [comment]
Company and Manager are stored in a different place than Core Properties in the XML files within the PPTX. Core properties are in "docProps/core.xlm" while Company and Manager are in "docProps/app.xml", which is not processed by the python-pptx library.

So, to get and edit those values the library must be updated to read "app.xml".

**Comment by MartinPacker** [comment]
The question becomes how to read and write an arbitrary (XML) file in the bundle that is a .pptx file.

**Comment by Sustoja** [comment]
You are absolutely correct. Here it is how I did it for my particular need (removing metadata from Office & PDF files): https://github.com/Sustoja/LimpiarMetadatos

More specifically, this is the function to access the extended metadata within the Office file:

EXTENDED_PROPERTIES = ['Company', 'Manager']

def _delete_extended_properties(unzipped_dir: Path) -> None:
    """ Removing 'Company' and 'Manager' metadata from app.xml. """

    try:
        app_xml_path = unzipped_dir / "docProps" / "app.xml"
        namespaces = {'ns': 'http://schemas.openxmlformats.org/officeDocument/2006/extended-properties'}

        tree = ET.parse(app_xml_path)
        root = tree.getroot()

        for tag in EXTENDED_PROPERTIES:
            element = root.find(f".//ns:{tag}", namespaces)
            if element is not None:
                root.remove(element)

        tree.write(app_xml_path, encoding="utf-8", xml_declaration=True)

    except Exception:
        raise RuntimeError("Unable to remove extended properties.")
